style.css

```diff
@@ -0,0 +1,90 @@
+body {
+    display: flex;
+   background-color:darkblue;
+   color: white;
+   justify-content: center;
+   align-items: center;
+   flex-direction: column;
+
+}
+#title {
+    text-align: center;
+    font-size: 25px;
+
+}
+#description {
+    text-align: center;
+    margin: 5px 0 15px;
+
+}
+form {
+    width: 60vw;
+    height: auto;
+    padding: 15px;
+    background-color: purple;
+    display: flex;
+    flex-direction: column;
+}
+input[type = "text"], input[type = "email"], input[type = "number"] {
+    display: block;
+    width: 100%;
+    padding: 12px 5px;
+    box-sizing: border-box;
+    margin : 12px 0;
+    font-size: 12px;
+    color: black; 
+}
+.container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+
+}
+.select {
+    display: flex;
+    flex-direction: column;
+
```

Comment:
Please can you manage a more evenly spaced-out code? when writing is CSS there should be one space after the element, ( .select { ) and 1 line space when ending a curly bracket. 

This will ensure, that when you add linters to your project it will not flag up as errors.